src/components/Schools/ClassTabs.js

```diff
@@ -10,7 +10,7 @@ const ClassTabs = ({classList}) => {
     const [isOpen, setIsOpen] = useState(false);
 
     const handleOpenClose = () =>{
-        if(isOpen == true){
```

Comment:
This has changed the logic from what it was before

It should just be `setIsOpen(!isOpen)`

Reply:
done